.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+- [ ] This has been discussed in issue #..., or if not, please tell us why
+  here.
+- [ ] A short description of what this do, why it is needed; if it is long
+  enough, it should probably be discussed in an issue first.
+- [ ] This PR contains test cases, if meaningful.
+- [ ] A reviewer from the core maintainer team has been assigned for this PR.
+  If you don't know who could review this, please indicate so and/or ping
+  `bnjbvr`. The list of suggested reviewers on the right can help you.
+- [ ] PR's author adheres to the [code of
+  conduct](https://github.com/CraneStation/cranelift/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
I wonder if the code of conduct hammer shouldn't be brought to bear on every PR but instead on departures from the code of conduct. It might be less hammer-like to add something like the following as a note (not a checklist item) on the end: Please ensure all communication adheres to the code of conduct.

Reply:
I agree.